src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.Properties.cs

```diff
@@ -91,7 +91,7 @@ public string Text
 		Windows.UI.Xaml.DependencyProperty.Register(
 			"TextMemberPath", typeof(string),
 			typeof(global::Windows.UI.Xaml.Controls.AutoSuggestBox),
-			new FrameworkPropertyMetadata(defaultValue: null)
+			new FrameworkPropertyMetadata(defaultValue: string.Empty)
```

Comment:
Hmm.. It looks like most controls in MUXC will have null default value but the setter does `?? ""`.
I'll need to adjust the analyzer for that and fix the PR.

Thanks @MartinZikmund for catching that! 🎉